std/shared/src/main/scala/cats/effect/std/MapRef.scala

```diff
@@ -119,7 +131,7 @@ object MapRef extends MapRefCompanionPlatform {
       seq: NonEmptySeq[Ref[F, Map[K, V]]]
   ): MapRef[F, K, Option[V]] = {
     val array = seq.toSeq.toArray
-    val shardCount = seq.size.toInt
+    val shardCount = seq.length
```

Comment:
Sorry to be a broken record, we should really prefer `array.length` here. Calculating the length for an arbitrary seq may be O(n) e.g. need to iterate over the entire linked list.
```suggestion
    val shardCount = array.length
```

Reply:
done